sdk/webpubsub/azure-messaging-webpubsub-client/src/test/java/com/azure/messaging/webpubsub/client/ClientTests.java

```diff
@@ -42,18 +42,15 @@ public void testClientState() {
 
```

Comment:
Have we run the live tests? Guessing we need to run the live tests and re-record to make sure the seriaization is expected. Add @weidongxu-microsoft @srnagar  for suggestion.

Reply:
I've tested azure-messaging-webpubsub-client, the only problem is https://github.com/Azure/azure-sdk-for-java/pull/40718/files#r1646838102

After this change all passes.

Reply:
I will try make a PR to switch azure-messaging-webpubsub-client away from CONNECTION_STRING.